src/status_im/chat/models/input.cljs

```diff
@@ -112,8 +115,19 @@
                 (chat-input-clear :input-ref)
                 (process-cooldown)))))
 
-(fx/defn send-sticker-fx
-  [{:keys [db] :as cofx} {:keys [hash pack]} current-chat-id]
+(fx/defn send-image
+  [{{:keys [current-chat-id] :as db} :db :as cofx}]
+  (let [image-path (get-in db [:chats current-chat-id :metadata :sending-image :uri])]
+    (fx/merge cofx
+              {:db (update-in db [:chats current-chat-id :metadata] dissoc :sending-image)}
+              (when-not (string/blank? image-path)
+                (chat.message/send-message {:chat-id      current-chat-id
+                                            :content-type constants/content-type-image
+                                            :image-path   (string/replace image-path #"file://" "")
```

Comment:
Do we need the `file://` at all? otherwise we could strip it as soon as it's created.